front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,168 @@
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <input type="text" id="genre" value={genre} ref={refGenre} readonly /><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="total_episodes">Total Episodes: </label>
+      <input type="text" id="total_episodes" value={total_episodes} ref={refTotalEpisodes} readonly /><br /><br />
+      <label for="isMovie">Is it a movie? </label>
+      <input type="text" id="isMovie" value={is_movie} ref={refIsMovie} readonly /><br /><br />
+    </div>
+  )
+}
+
+/*the component calls other components and displays comprehensive info about the show; 
+values passed as parameters to the main components are generated by the mock data API */
+
+const IndividualShow = (props) => {
+
+  const refTitle = useRef();
+  const refCover = useRef();
+  const returnToShows = (() => {
+    window.location.href = '/my-shows/'
+  });
+  const addToWatched = (() => {
+
+    window.location.href = '/my-shows/'
+  });
+
+
+  let [show, setShow] = useState([]);
+
+  useEffect(() => {
+    //temporary variable to be replaced
+    let showInfo = [];
+    let show_id = "54";
+    //axios.get(`https://my.api.mockaroo.com/shows/${show.id}.json?key=3010e030`)
+    axios.get(`https://my.api.mockaroo.com/shows/${show_id}.json?key=3010e030`)
+      .then((response) => {
+        showInfo.push(response.data);
+        console.log(showInfo);
+        console.log(show);
+        setShow(showInfo);
+      })
+  }, [])
+
+
+  return (
+    <>
+      <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
+      <div className="header"><h2>Show Page</h2></div>
+
+      <div class="show_content">
+        <fieldset class="main">
+          <div class="show-details">
+            <fieldset >
+              {show.map(s => ( //display name of the show
+                <h3 id="title" value={s.name} ref={refTitle}>{s.name}</h3>
+              ))}
+              <button onClick={(e) => returnToShows()}>
+                Return to Shows
+      </button>
+              <button onClick={(e) => addToWatched()}>
+                Add to In Progress or Watched Shows
+      </button>
+              <ProgressData season="5" episode="7" />
+              <br /><br />
+              <PlatformData />
+              <br /><br />
+              <div class="show_content">
+                {show.map(s => ( //display general info about the show
+                  <Description genre={s.genres} description={s.description} total_episodes={s.episodes} is_movie={s.isMovie} />
+                ))}
+
+                <br />
+              </div>
+            </fieldset>
+
+          </div>
+          {show.map(s => ( //display cover image 
+            <img id="cover" src={s.coverPhoto} alt="" ref={refCover}></img>
+          ))}
+          <div id="clear"></div>
```

Comment:
What is this for?

Reply:
this retrieves cover image from the API based on the show id

Reply:
Ah, I meant the `<div id="clear">`

Reply:
div id="clear” is for aligning the text